docs/manual/deployment.md

```diff
@@ -43,13 +43,14 @@ deployment.  For short, we will call this the "tree", to
 distinguish it from the concept of a deployment.
 
 First, the tree must include a kernel stored as
-`/boot/vmlinuz-$checksum`.  The checksum should be a SHA256 hash of
-the kernel contents; it must be pre-computed before storing the kernel
-in the repository.  Optionally, the tree can contain an initramfs,
-stored as `/boot/initramfs-$checksum`.  If this exists, the checksum
-must include both the kernel and initramfs contents.  OSTree will use
-this to determine which kernels are shared.  The rationale for this is
-to avoid computing checksums on the client by default.
+`/boot/vmlinuz(-.*)?-$checksum`.  The checksum should be a SHA256 hash
+of the kernel contents; it must be pre-computed before storing the
+kernel in the repository.  Optionally, the tree can contain an
+initramfs, stored as `/boot/initramfs(-.*)?-$checksum`.  If this
```

Comment:
While you're amending this, might as well mention all of this content can now be in either `/usr/lib/ostree-boot/` or `/boot`.